pkg/server/server.go

```diff
@@ -70,10 +69,6 @@ func NewSSIServer(shutdown chan os.Signal, cfg config.SSIServiceConfig) (*SSISer
 	engine.GET(ReadinessPrefix, router.Readiness(ssi.GetServices()))
 
 	// swagger
-	doc.SwaggerInfo.Version = cfg.SVN
-	doc.SwaggerInfo.Description = cfg.Desc
-	doc.SwaggerInfo.Host = cfg.Server.APIHost
```

Comment:
it's useful to set these dynamically based on config

Reply:
Can you elaborate on why this is useful? 

There are a couple of reasons why I think it's more useful to not have this here. 
1. This fixes #522
2. When setting this dynamically there ends up being a mismatch between the content of the `swagger.yaml` file and what's hosted.
3. The semantic versioning is something that's defined per PR, and shouldn't be changed in config files. In fact, it's required by the openapi spec.
4. Having this be dynamic requires a 2-step generation.

Additionally, I wasn't able to tell how these lines are achieving any visible changes.

Reply:
If we can figure out a way to maintain these properties - and have them up to date with our config - then no problems from me.

I believe there's another issue that is tangential to this as well. We need to distinguish between "service information" - like routes and versions - and user specified config.